pkg/engine/harpoon.go

```diff
@@ -381,16 +377,6 @@ func (hc *HarpoonConfig) processConfig(ctx context.Context, target *Target) {
 		return
 	}
 	hc.restartHarpoon = restart
-	mo := &SingleMethodObj{
-		Conn:   hc.conn,
-		Method: configMethod,
-		Target: target,
-	}
-	if err := hc.EngineMethod(ctx, mo, defaultConfigPath, nil); err != nil {
-		klog.Warningf("Target: %s error updating config targets, will retry next run: %v", err)
-		return
-	}
-
```

Comment:
this filetransfer is not needed - whatever is passed to podman at /path/to/configdir:/opt/mount is where the config & backup config will be